### PR TITLE
Feature - use GitHub draft release with titles

### PR DESCRIPTION
**Pull request by stijnmoreels**
Relates to https://github.com/arcus-azure/arcus/issues/89

### CODE REVIEW

File: github/create-release.yml [hunk 1]
```diff
@@ -33,13 +33,34 @@ steps:
       Write-Host "GitHub release is pre-release because of $Env:TAG"
       Write-Host "##vso[task.setvariable variable=isPreRelease]true"
       Write-Host "##vso[task.setvariable variable=compareWith]lastRelease"
+      Write-Host "##vso[task.setvariable variable=releaseNotes]$Env:RELEASE_NOTES"
+      Write-Host "##vso[task.setvariable variable=isDraft]false"
```
Comment (tomkerkhove) [inline github/create-release.yml:37]: Why are we setting it to false? 🤔

Comment (tomkerkhove) [reply github/create-release.yml:37]: I would always use draft releases - For both full & preview releases

Comment (tomkerkhove) [reply github/create-release.yml:37]: Although you could argue preview is not required, fair point.

**Review by tomkerkhove — DISMISSED** [review]

File: github/create-release.yml (hunk 1 above)
Comment (stijnmoreels) [reply github/create-release.yml:37]: Yeah, it was bc we only have added the titles to the full releases and so the current setup of pre-releases is still 'complete' in the sense that only having `Install-Package -Name ...` is enough as release notes.

**Review by tomkerkhove — APPROVED** [review]

**Pull request merged**